src/rust/ide/src/view/project_name.rs

```diff
@@ -58,18 +104,44 @@ impl ProjectName {
             project_name.text_field.set_content(&new_name);
             project_name.setup_center_alignment();
         });
+
+        let network      = &self.network;
+        let project_name = self.clone_ref();
+        frp::extend! { network
+            def _foo = self.main_area.events.mouse_over.map(f_!(project_name.highlight_text()));
+            def _foo = self.main_area.events.mouse_out .map(f_!(project_name.darken_text()));
```

Comment:
`_foo` is bad name. In such case use eval. And use animations - they will change this frp net